docs/sphinx/source/whatsnew/v0.11.1.rst

```diff
@@ -30,6 +30,10 @@ Enhancements
 * Added function for calculating wind speed at different heights,
   :py:func:`pvlib.atmosphere.windspeed_powerlaw`.
   (:issue:`2118`, :pull:`2124`)
+* Implemented closed-form solution for alpha in detect_clearsky, obviating
+  the call to scipy.optimize that was prone to runtime errors and minimizing
+  computation, :py:func:`pvlib.clearsky.detect_clearsky`. Resolves :issue:`2712`.
```

Comment:
Link the function in the first line and then no need to repeat it here at the end as (I think?) we only do that if it is a new function being added
```suggestion
* Implemented closed-form solution for alpha in
  :py:func:`pvlib.clearsky.detect_clearsky`, obviating
  the call to scipy.optimize that was prone to runtime errors and minimizing
  computation. (:issue:`2171`, :issue:`2216`, :pull:`2217`).
```